sections/0-abstract.tex

```diff
@@ -1,7 +1,6 @@
 \begin{abstract}
-    The primary goal of the \sunpy Project is to facilitate and promote the use and development of a community-led, free and open-source software for solar physics based on the scientific \python environment. 
-    This goal is primarily achieved through the development of the \sunpypkg core package which aims to provide foundational capabilities to the community. 
-    The Project also supports a number of  affiliated package which depend on the core package. 
+    The \sunpy Project facilitates and promotes the use and development of several community-led, free, and open-source data analysis software packages for solar physics based on the scientific \python environment. 
+    The project achieves this goal by developing and maintaining the \sunpypkg core package and supporting an ecosystem of affiliated packages.
```

Comment:
I had used the word `primarily` to suggest that we do other things (like run tutorials or advocacy). This wording suggests we do nothing else.

Reply:
I added the word primarily back in.